src/app/layout.tsx

```diff
@@ -0,0 +1,68 @@
+import Script from "next/script";
+import { Footer } from "../components/footer";
+import { Header } from "../components/header";
+import "./globals.css";
+import { Roboto_Flex, Roboto_Mono } from "next/font/google";
+import { Metadata } from "next";
+
+// Loads fonts: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts
+export const robotoFlex = Roboto_Flex({ subsets: ["latin"], display: "swap" });
+export const robotoMono = Roboto_Mono({ subsets: ["latin"], display: "swap" });
```

Comment:
Can you check the PageSpeed score? https://pagespeed.web.dev/
We can drop custom fonts if that will help with score, or optimize the build somehow.

Reply:
I've been testing with Chrome's LightHouse and getting 90+
Will test with pagespeed too

Reply:
Oh, pagespeed and lighthouse are the same tool
You can take a look here: https://pagespeed.web.dev/analysis/https-devchecklists-rangelfinal-vercel-app/6bdw399fqh?form_factor=mobile
It's main complaints are:
- console errors (Errors when trying to get avatars from github, are auto handled by js)
- We don't provide next-gen image formats
- There are a few places where contrast of background to foreground is not enough for accessibility
- Github avatars should be chached for longer
- Some images should be higher res

Reply:
Please bring these results to next Dev Weekly at Vinta! You got better results than what we had before with the Python-based sitegen.